mad-dog/package.json

```diff
@@ -30,9 +30,7 @@
     "lodash": "^4.17.15",
     "logform": "^2.1.2",
     "moment": "^2.29.1",
-    "node-fetch": "^2.6.0",
```

Comment:
I moved around some dependencies from mad-dog to service-commands as part of the refactor to move `random*` APIs that are shared by both seed CLI and mad dog integration tests to service-commands utils.